src/common/tools/gen_wpt_cts_html.ts

```diff
@@ -49,8 +50,8 @@ const [
     const entries = (await listing) as TestSuiteListingEntry[];
     const lines = entries
       // Exclude READMEs.
-      .filter(l => l.path.length !== 0 && !l.path.endsWith('/'))
-      .map(l => '?q=webgpu:' + l.path);
+      .filter(l => 'description' in l)
```

Comment:
should this be `readme` in l ?

Reply:
it's filtering on `'description' in l` which is equivalent to `!('readme' in l)` which would be clearer